resources/views/index.blade.php

```diff
@@ -11,20 +11,21 @@
         </div>
         <!-- Source: -->
         <!-- http://openclipart.org/detail/34273/tango-system-lock-screen-by-warszawianka -->
-        <img src="/static/img/locked-screen.png" />
+        <img src="/img/locked-screen.png" />
       </div>
       <div class="span6">
 
         <?php if (isset($errorMessage)): ?>
           <div class="alert alert-error">
             <button type="button" class="close" data-dismiss="alert">&times;</button>
-            <span id="error_message">${errorMessage}</span>
+            <span id="error_message"><?=$errorMessage?></span>
```

Comment:
Same here, use blade to display values, not php blocks. https://laravel.com/docs/5.2/blade#displaying-data